src/zabapgit_object_samc.prog.xml

```diff
@@ -10,13 +10,6 @@
     <RLOAD>E</RLOAD>
     <UCCHECK>X</UCCHECK>
    </PROGDIR>
-   <TPOOL>
-    <item>
-     <ID>R</ID>
-     <ENTRY>Include ZABAPGIT_OBJECT_SAMC</ENTRY>
-     <LENGTH>28</LENGTH>
-    </item>
-   </TPOOL>
```

Comment:
Looks like the include description was lost, if you enter it again manually and push it should be okay(also relevant for the other include)